README.md

```diff
@@ -125,14 +125,14 @@ See [CONTRIBUTING.md](CONTRIBUTING.md)
     - [Browser testing suite provided by](#browser-testing-suite-provided-by)
   - [License](#license)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- END doctoc generated TOC please keep comment here to allow auto-update -->
```

Comment:
Since this is used by doctoc and not seen by users, this should probably be left without the hyphen.